### ISSUE TITLE
[Feature Request] Customizable domains

### ISSUE THREAD

**Issue by Sirozha1337** [webhooksite/webhook.site#130]
It would have been great to have customizable domains like this: {guid}.webhook.site, with webhooks created inside this domain looking like this {guid}.webhook.site/{path}. That way you can create a domain for every service you're trying to mock and then create mocked paths inside each domain.

**Comment by agardnerIT** [comment]
Agreed. Also, when running on k8s, the domain comes out as `http://localhost:8084/UUID` which should probably be configurable to be something like `https://webhook.webhook.service.cluster.local:8084/UUID`
